src/cms/templates/users/admin/user.html

```diff
@@ -83,6 +83,11 @@ <h3 class="heading font-bold text-white">{% trans 'Extended Settings' %}</h3>
                     {% render_field user_form.is_active %}
                 </div>
 
+                <div class="py-4 border-b solid border-gray-200">
+                    <label for="id_expert_mode" class="font-bold mr-2 cursor-pointer">{% trans 'Expert mode' %}</label>
+                    {% render_field user_profile_form.expert_mode %}
```

Comment:
I'd like to have an explanation here what features exactly are enabled with expert mode (and in the region user form as well).

Reply:
Alright, I added an explanation paragraph